core/src/test/scala/unit/kafka/log/LogValidatorTest.scala

```diff
@@ -73,13 +80,18 @@ class LogValidatorTest {
   }
 
   private def checkMismatchMagic(batchMagic: Byte, recordMagic: Byte, compressionType: CompressionType): Unit = {
-    assertThrows[InvalidRecordException] {
+    try {
```

Comment:
nit: its more convenient to use `assertThrow`.

Reply:
ack